examples/get_by_name.rs

```diff
@@ -0,0 +1,54 @@
+use anyhow::{anyhow, Result};
+use scylla::transport::session::Session;
+use scylla::SessionBuilder;
+use std::env;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    tracing_subscriber::fmt::init();
+    let uri = env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1".to_string());
+
+    println!("Connecting to {} ...", uri);
+
+    let session: Session = SessionBuilder::new().known_node(uri).build().await?;
+
+    session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}", &[]).await?;
+
+    session
+        .query(
+            "CREATE TABLE IF NOT EXISTS ks.hello (pk int, ck int, value text, primary key (pk, ck))",
+            &[],
+        )
+        .await?;
+
+    session
+        .query(
+            "INSERT INTO ks.hello (pk, ck, value) VALUES (?, ?, ?)",
+            (3, 4, "def"),
+        )
+        .await?;
+
+    session
+        .query(
+            "INSERT INTO ks.hello (pk, ck, value) VALUES (1, 2, 'abc')",
+            &[],
+        )
+        .await?;
+
+    let query_result = session
+        .query("SELECT pk, ck, value FROM ks.hello", &[])
+        .await?;
+    let (ck_idx, _) = query_result
+        .get_spec_by_name("ck")
```

Comment:
The `get_spec_by_name` _function name_ looks confusing to my 👀. Perhaps something like `get_column_spec` is more readable?

Reply:
Sure. I added `_by_name` in case we ever want to get the spec by index, but for that we can just use the vector directly. I'll change to `get_column_spec`